slack/web/async_slack_response.py

```diff
@@ -156,6 +168,12 @@ def get(self, key, default=None):
         Returns:
             The value from data or the specified default.
         """
+        if isinstance(self.data, bytes):
+            raise ValueError(
+                "As the response.data is binary data, this operation is unsupported"
+            )
+        if self.data is None:
```

Comment:
get method should return `None` even if the data is `None` in the same manner with dictionary objects.